test/SlowTests/Issues/RavenDB-23136.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Utils;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Client.Documents.Smuggler;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_23136 : RavenTestBase
+    {
+        public RavenDB_23136(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenFact(RavenTestCategory.Smuggler | RavenTestCategory.Revisions)]
+        public async Task Import_Should_Create_Revisions_When_Configuration_Is_On(bool withRevisionsConfig)
+        {
+            DoNotReuseServer();
+            var files = new List<string>()
+            {
+                GetTempFileName(),
+                GetTempFileName(),
+                GetTempFileName()
+            };
+            try
+            {
+                await Import_Should_Create_Revisions_When_Configuration_Is_On_Internal(files, withRevisionsConfig);
+            }
+            finally
+            {
+                foreach (var f in files)
+                {
+                    File.Delete(f);
+                }
+            }
```

Comment:
no need for that, they should be deleted automatically

Reply:
Removed